web/src/components/SearchInput.tsx

```diff
@@ -63,6 +63,7 @@ export class SearchInput extends React.PureComponent<PropsType> {
       <Spacer space={spaceSearch}>
         <Wrapper>
           <SearchIcon />
+          {/* eslint-disable-next-line styled-components-a11y/no-autofocus */}
```

Comment:
a11y doesn't recommend auto-focus at all ;)

Reply:
Perhaps you could add a comment why this is fine in our case?